src/server.ts

```diff
@@ -140,13 +139,24 @@ void cronStarter(bot);
 
 // webhook
 if (env.WEBHOOK_ENABLED) {
-  const server = Fastify();
-
-  server.post(`/${env.TOKEN}`, webhookCallback(bot, "fastify"));
-  server.setErrorHandler(errorHandler);
-  server.listen({ port: env.WEBHOOK_PORT }, async () => {
-    await bot.api.setWebhook(env.WEBHOOK_URL + env.TOKEN);
-  });
+  env.WEBHOOK_URL = "https://3ac4-84-54-66-241.ngrok-free.app/";
+  env.WEBHOOK_PORT = 8000;
+
+  const secretToken = env.TOKEN.split(":")[1];
```

Comment:
can you explain what's going on in here pls

Reply:
we use token as auth

Reply:
ok